src/mawpy/steps/incremental_clustering.py

```diff
@@ -345,41 +338,40 @@ def _run(args):
     return df_by_user
 
 
-def _divide_chunks(user_id_list, n):
-    user_id_chunks = []
-    for i in range(0, len(user_id_list), n):  # looping till length usernamelist
-        user_id_chunks.append(user_id_list[i: i + n])
-    return user_id_chunks
-
-
-def incremental_clustering(input_file, output_file, spatial_constraint, dur_constraint):
+def incremental_clustering(input_file: str, output_file: str, spatial_constraint: float, dur_constraint: float) -> None:
+    # this_lock = Lock()  # thread locker
```

Comment:
looks like the lock was not used or is not needed? If so we can remove the commented code. Can we also remove `init(...)` in line 32?

Reply:
done